ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/TemplateConfigurationFactory.kt

```diff
@@ -28,9 +25,9 @@ internal object TemplateConfigurationFactory {
         val createPackageResult =
             PackageConfigurationFactory.createPackageConfiguration(
                 variableDataProvider = variableDataProvider,
-                packages = packages,
+                availablePackages = availablePackages,
                 activelySubscribedProductIdentifiers = activelySubscribedProductIdentifiers,
-                filter = packageIds,
+                packageIdsInConfig = paywallData.config.packages,
```

Comment:
I wonder if we should rename the property `packages` to `packageIds`, even if the backend response remains the same... But that can be done separately if needed.

Reply:
That will make things clearer. I will open a PR